### PR TITLE
Add MOT and MOTS data format docs

### PR DESCRIPTION
**Pull request by vinnamkim**
### Summary

- Ticket no. 106351
- Same as title

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [x] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [x] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [x] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: docs/source/docs/explanation/formats/mot.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Multiple Object Tracking (MOT)
+
+## Format specification
+
+The [Multiple Object Tracking (MOT) challange dataset](https://arxiv.org/pdf/1906.04567.pdf) provides bounding box tracking data for multiple objects within image sequences.
```
Comment (sooahleex) [inline docs/source/docs/explanation/formats/mot.md]:
```suggestion
The [Multiple Object Tracking (MOT) challenge dataset](https://arxiv.org/pdf/1906.04567.pdf) provides bounding box tracking data for multiple objects within image sequences.
```

File: docs/source/docs/explanation/formats/mot.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+Supported annotation attributes:
+- `track_id` (int) - Unique ID assigned to an object within a trajectory
+- `visibility` (float) - Visibility ratio of each bounding box due to occlusion by another static or moving object, or due to image border cropping
+- `occluded` (boolean) - `True` if `visibility < occlusion_threshold`, otherwise `False`
+- `ignored` (boolean) - `True` if the confidence score of bounding box is zero, otherwise `False`
+- `score` (float) - Confidence score of bounding box
+
+## Import MOT dataset
+
+You can download the MOT challange dataset [here](https://motchallenge.net).
```
Comment (sooahleex) [inline docs/source/docs/explanation/formats/mot.md]:
```suggestion
You can download the MOT challenge dataset [here](https://motchallenge.net).
```

File: docs/source/docs/explanation/formats/mot.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+``` bash
+datum project create
+datum project import --format mot <path/to/dataset>
+```
+
+It is possible to specify project name and project directory. Run
+`datum project create --help` for more information.
+
+The MOT chllange dataset directory should have the following structure:
```
Comment (sooahleex) [inline docs/source/docs/explanation/formats/mot.md]:
```suggestion
The MOT challenge dataset directory should have the following structure:
```

File: docs/source/docs/explanation/formats/mots.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
+# Multiple Object Tracking and Segmentation (MOTS)
+
+## Format specification
+
+The [Multiple Object Tracking and Segmentation (MOTS) challange dataset](https://arxiv.org/pdf/1906.04567.pdf) provides a pixel-level segmentation masks for multiple objects within image sequences.
```
Comment (sooahleex) [inline docs/source/docs/explanation/formats/mots.md]:
```suggestion
The [Multiple Object Tracking and Segmentation (MOTS) challenge dataset](https://arxiv.org/pdf/1906.04567.pdf) provides a pixel-level segmentation masks for multiple objects within image sequences.
```

**Review by sooahleex — DISMISSED** [review]
Looks good to me. I just found minor typo.

**Review by cih9088 — APPROVED** [review]

**Review by sooahleex — APPROVED** [review]

**Pull request merged**